packages/client-core/src/admin/components/Setting/Authentication.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import { Grid, Paper, Button, Typography } from '@material-ui/core'
+import InputBase from '@material-ui/core/InputBase'
+import { useStyles } from './styles'
+import { Divider } from '@material-ui/core'
+
+const Account = () => {
+  const classes = useStyles()
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+  }
+  return (
+    <div className={`${classes.root} ${classes.container}`}>
+      <Typography component="h1" className={classes.settingsHeading}>
+        {' '}
+        Authentication
+      </Typography>
+
+      <form autoComplete="off" onSubmit={handleSubmit}>
+        <Grid container spacing={3}>
+          <Grid item xs={6} sm={4}>
+            <label> SERVICE</label>
+            <Paper component="div" className={classes.createInput}>
+              <InputBase name="service" style={{ color: '#fff' }} className={classes.input} />
+            </Paper>
+            <label>SECRETE</label>
```

Comment:
This and several other instances should be Secret, not Secrete (drop the ending 'e')